doc/generate.py

```diff
@@ -3,23 +3,35 @@
 import sys
 import os
 from contextlib import suppress
-
 import toucan_connectors
-
+from importlib import import_module
 
 def doc_or_empty(klass):
     with suppress(AttributeError):
         return klass.__doc__.strip()
 
+def clean_type_str(field_str):
```

Comment:
Do we agree that this function does not concern the module importation issue ? Or do I miss something ?

Reply:
You're right, I also changed the cleaning of the string retrieved from class definition. 
I'll add that in the change description.